src/components/tag/Tag.stories.mdx

```diff
@@ -1,40 +1,82 @@
 import { Meta, Story, Preview } from "@storybook/addon-docs/blocks";
 import { Tag } from ".";
-import { GnuiContainer } from "../container";
+import { Heading } from "../heading";
+import { Flex } from "../container";
+import { Breadcrumbs } from "../breadcrumbs";
 
 <Meta title="Components/Tag" component={Tag} />
 
 # Tag
 
+- `color`: `string` – _optional_, background color of Tag by default background is `theme.colors.lights[2]`
+- `colorText`: `string` – _optional_, text color of Tag by default text color is `theme.colors.primary`
+- `text`: `string` — _optional_, Tag content
+- `icon`: `string` — optional, You can use icon prop to add it to the tag. List of available icons you can find [here](?path=/docs/components-svgicon--ui)
```

Comment:
You can use icon prop to add it to the tag. List of available icons you can find [here](?path=/docs/components-svgicon--ui)

we can write it more concisely, e.g.:
"icon to be placed inside the tag,  [list of available icons](?path=/docs/components-svgicon--ui)"